src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/LaunchProfilesDebugLaunchProvider.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
+using Microsoft.VisualStudio.Debugger.UI.Interfaces.HotReload;
```

Comment:
What types from Debugger.UI.Interfaces do we need?

Reply:
`IHotReloadOptionService`

Reply:
I see, it's forwarded to contracts: 

```
// Forwarded from Microsoft.VisualStudio.Debugger.UI.Interfaces.dll for dependencies which do not want to
// import it.
namespace Microsoft.VisualStudio.Debugger.UI.Interfaces.HotReload
```